internal/tui/components/app/app.go

```diff
@@ -510,6 +510,23 @@ func (m *Model) handleKeyMsg(msg tea.KeyMsg) tea.Cmd {
 				}
 			}
 
+		case key.Matches(msg, keymap.KeyMap.GarbageCollect):
+			err := m.client.System().GarbageCollect()
```

Comment:
If this action hangs for one reason or another, it will hang the whole wander app (unable to navigate, quit, etc). This is why bubbletea runs async actions in Cmds, which run in non-deterministic order in goroutines outside the main application event loop. See https://github.com/charmbracelet/bubbletea/tree/master/tutorials/commands/ for details :)

Reply:
Got it. Will keep that in mind.